src/clusterfuzz/_internal/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -3353,6 +3353,18 @@ def test_dns_resolution(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_pysecsan_command_os_system(self):
+    """Test PySecSan command injection bug in os.system"""
+    data = self._read_test_data('pysecsan_command_os_system.txt')
+    expected_type = 'PySecSan'
```

Comment:
Can we make this a bit more detailed about what exact type it is?

Reply:
I added it as a "PySecSan" type to allow for more experimentation in PySecSan (as in different types of bug detectors) without having to modify Clusterfuzz whenever trying one -- wdyt?

Reply:
For the future we will want more detailed crash types, but if this is to experiment quickly this sounds fine.

Reply:
Sounds good